pkg/tesla/telemetry/codegen/codegen.go

```diff
@@ -315,6 +328,12 @@ var conversions = map[string]map[string]string{
 	},
 }
 
+var parsers = map[string]string{
```

Comment:
Is there any nice way around making these maps for parsing and converting units?